llvm/test/CodeGen/X86/pr108731.ll

```diff
@@ -0,0 +1,40 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=x86_64-gnu-unknown -mcpu=znver3 | FileCheck %s
```

Comment:
Make test more generic - replace znver3 with x86-64-v3 and add a x86-64-v2 test for case without BMI1 support:
```
RUN: llc < %s -mtriple=x86_64-- -mcpu=x84-64-v2 | FileCheck %s --check-prefixes=CHECK,NOBMI
RUN: llc < %s -mtriple=x86_64-- -mcpu=x84-64-v3 | FileCheck %s --check-prefixes=CHECK,BMI
```

Reply:
Didn't know about `x86-64-v3` or `x86-64-v2`, thank you!